test/functional/test_runner.py

```diff
@@ -58,102 +58,115 @@
 BASE_SCRIPTS = [
     # Scripts that are run by the travis build process.
     # Longest test should go first, to favor running tests in parallel
-    'wallet_hd.py',
-    'wallet_backup.py',
-    # vv Tests less than 5m vv
-    'feature_block.py',
-    'rpc_fundrawtransaction.py',
-    'p2p_compactblocks.py',
-    'feature_segwit.py',
+    'feature_fee_estimation.py',
     # vv Tests less than 2m vv
-    'wallet_basic.py',
-    'wallet_labels.py',
+    'feature_block.py',
```

Comment:
to be honest, I don't think the reordering matters. for example, feature_block was the third test previously and is now the second test. I think it is most important to have the longest running test as the first test. The order of the remaining ones shouldn't matter much if they are reasonably sorted by run time.